package.json

```diff
@@ -35,7 +35,7 @@
     "js-yaml": "^3.13.1",
     "jsonc-parser": "^2.2.1",
     "request-light": "^0.2.4",
-    "vscode-json-languageservice": "^3.6.0",
+    "vscode-json-languageservice": "^3.6.0 <3.8.0",
```

Comment:
This would not allow `3.8.0` only `3.6.0 - 3.7.x`.
If you are trying to allow 3.8.0 then you need `<=`, since you reference a bug in 3.10.
It might be easier to use the dash instead of the carat and less then sign. `3.6.0 - 3.8.0`, if you intend it to go 3.8.

Reply:
the `yarn.lock` was pinned back on 3.6.0. Trying the initial pin lead to a `tsc` fail: https://github.com/redhat-developer/yaml-language-server/runs/1350938321

I did a weak bisection and came up with this version... who knows, maybe they broke semver and it works somewhere into the 3.7 line, but this seemed fine.

Also can't keep track of all their turing-complete spec language, but will update to `-`...

Reply:
pushed this change on cb301a5

Reply:
oh, right, that's wrong, because now it _will_ install 3.8, and presumably break... grumble

Reply:
or not break... thoroughly confused... locally, having `3.8` does break on:

```bash
$> yarn && yarn build

yarn install v1.22.10
[1/5] Validating package.json...
[2/5] Resolving packages...
success Already up-to-date.
Done in 0.25s.
yarn run v1.22.10
$ yarn clean && yarn lint && yarn compile && yarn build:libs
$ rimraf out/server && rimraf lib
$ eslint -c .eslintrc.js --ext .ts src test
$ tsc -p .
src/languageservice/services/yamlCompletion.ts:72:11 - error TS2322: Type 'import("~/node_modules/vscode-json-languageservice/lib/umd/jsonLanguageTypes").Thenable<import("~/node_modules/vscode-json-languageservice/node_modules/vscode-languageserver-types/lib/umd/main").Co...' is not assignable to type 'import("~/node_modules/vscode-json-languageservice/lib/umd/jsonLanguageTypes").Thenable<import("~/node_modules/vscode-languageserver-types/lib/umd/main").CompletionItem>'.
  Type 'import("~/node_modules/vscode-json-languageservice/node_modules/vscode-languageserver-types/lib/umd/main").CompletionItem' is not assignable to type 'import("~/node_modules/vscode-languageserver-types/lib/umd/main").CompletionItem'.
    Types of property 'textEdit' are incompatible.
      Type 'TextEdit | InsertReplaceEdit' is not assignable to type 'TextEdit'.
        Property 'range' is missing in type 'InsertReplaceEdit' but required in type 'TextEdit'.

72           return resolver;
             ~~~~~~~~~~~~~~~~

  node_modules/vscode-languageserver-types/lib/umd/main.d.ts:486:5
    486     range: Range;
            ~~~~~
    'range' is declared here.


Found 1 error.
```

so i'll stand by 0035ab2 and don't have any other fiddling planned. Feel free to squash, if merging, sorry for the noise.